src/migration-wizard/steps/discovery/EnhancedDownloadButton.tsx

```
@@ -0,0 +1,654 @@
+  Spinner,
+  Dropdown,
+  DropdownItem,
+  DropdownList,
+  MenuToggle,
+  MenuToggleElement,
+} from '@patternfly/react-core';
+import { createRoot } from 'react-dom/client';
+import { DownloadIcon } from '@patternfly/react-icons';
+import type { Source } from '@migration-planner-ui/api-client/models';
+
+import './DownloadPDFStyles.css';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)